Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Management/Driver.hs

```diff
@@ -1217,21 +1218,21 @@ getDriverStats merchantShortId opCity mbEntityId mbFromDate mbToDate requestorId
                   }
         _ -> throwError (InvalidRequest "Invalid date range: Dates must be empty, same day, or max 7 days apart.")
 
-    isAssociationBetweenTwoPerson :: (EsqDBFlow m r, MonadFlow m, CacheFlow m r) => DP.Person -> DP.Person -> m Bool
-    isAssociationBetweenTwoPerson requestedPersonDetails personDetails = do
-      case (requestedPersonDetails.role, personDetails.role) of
-        (DP.OPERATOR, DP.DRIVER) -> checkDriverOperatorAssociation personDetails.id requestedPersonDetails.id
-        (DP.OPERATOR, DP.FLEET_OWNER) -> checkFleetOperatorAssociation personDetails.id requestedPersonDetails.id
-        (DP.FLEET_OWNER, DP.DRIVER) -> checkFleetDriverAssociation requestedPersonDetails.id personDetails.id
-        _ -> return False
-
     calculateEarningsPerKm :: Meters -> HighPrecMoney -> HighPrecMoney
     calculateEarningsPerKm distance earnings =
       let distanceInKm = distance `div` 1000
        in if distanceInKm.getMeters == 0
             then HighPrecMoney 0.0
             else toHighPrecMoney $ roundToIntegral earnings `div` distanceInKm.getMeters
 
+isAssociationBetweenTwoPerson :: (EsqDBFlow m r, MonadFlow m, CacheFlow m r) => DP.Person -> DP.Person -> m Bool
+isAssociationBetweenTwoPerson requestedPersonDetails personDetails = do
+  case (requestedPersonDetails.role, personDetails.role) of
+    (DP.OPERATOR, DP.DRIVER) -> checkDriverOperatorAssociation personDetails.id requestedPersonDetails.id
```

Comment:
what about indirect association D <-> F + F <-> O? I think we should allow uploading docs also